pages/maintenance.tsx

```diff
@@ -0,0 +1,11 @@
+import Seo from 'components/Layout/Seo/Seo';
+
+const Maintenance = (): React.ReactElement => (
+  <>
+    <Seo title="Maintenance mode" />
+    <h1>Sorry, the service is unavailable</h1>
+    <p className="govuk-body"></p>
```

Comment:
I think it's worth still have some content. Maybe:

```suggestion
    <p className="govuk-body">You will be able to use the service from 8pm on Tuesday 24 May 2021.</p>
```